substrate-node/runtime/src/lib.rs

```diff
@@ -444,6 +446,34 @@ impl pallet_scheduler::Config for Runtime {
     type WeightInfo = ();
 }
 
+pub struct AuraAccountAdapter;
+use sp_runtime::ConsensusEngineId;
+
+impl FindAuthor<AccountId> for AuraAccountAdapter {
+    fn find_author<'a, I>(digests: I) -> Option<AccountId>
+    where
+        I: 'a + IntoIterator<Item = (ConsensusEngineId, &'a [u8])>,
+    {
+        if let Some(index) = pallet_aura::Module::<Runtime>::find_author(digests) {
+            let validator = pallet_session::Module::<Runtime>::validators()[index as usize].clone();
+            Some(validator)
+        } else {
+            None
+        }
+    }
+}
+
+parameter_types! {
+	pub const UncleGenerations: u32 = 0;
```

Comment:
wasn't this part of the modification in staking to fix the fact that some points were rewarded to the all zero pub key account? But since we are rewarding only Aura block producers here, I guess this is the correct thing to do.